src/Essentials/test/DeviceTests/Tests/SecureStorage_Tests.cs

```diff
@@ -223,5 +223,14 @@ await Parallel.ForEachAsync(Enumerable.Range(0, 100), async (i, _) =>
 				Assert.Null(fetched);
 			});
 		}
+
+		[Theory]
+		[InlineData("myKey", "value123")]
+		[InlineData("myKey", "")]
+		public async Task Get_Or_Set_Async_Returns_Value(string key, string value)
```

Comment:
The test is failing on Mac and Windows (empty value case):
<img width="371" alt="image" src="https://github.com/user-attachments/assets/5bf180cb-d6d6-4c20-a812-9fa67b3c1482" />

Reply:
Noted, addressing soon. I have a slight idea of what can be happening